helix-term/src/ui/mod.rs

```diff
@@ -155,11 +155,7 @@ pub fn file_picker(root: PathBuf, config: &helix_view::editor::Config) -> FilePi
         files,
         move |path: &PathBuf| {
             // format_fn
-            path.strip_prefix(&root)
-                .unwrap_or(path)
-                .to_str()
-                .unwrap()
-                .into()
```

Comment:
This can be hundreds of allocations on a large list. You should try to use to_str() then fallback.

Reply:
Would it allocate? `to_string_lossy` returns a `Cow<str>`

Reply:
Ah okay, then that's fine.